templates/gin/files/server.go

```diff
@@ -0,0 +1,147 @@
+package main
+
+import (
+	"net/http"
+	"fmt"
+	"database/sql"
+	_ "github.com/go-sql-driver/mysql"
+	psh "github.com/platformsh/config-reader-go/v2"
+	sqldsn "github.com/platformsh/config-reader-go/v2/sqldsn"
+	"github.com/gin-gonic/gin"
+)
+
+var db = make(map[string]string)
+
+func setupRouter() *gin.Engine {
+	// Disable Console Color
+	// gin.DisableConsoleColor()
+	r := gin.Default()
+
+	// Hello World
+	r.GET("/", func(c *gin.Context) {
+
+		config, err := psh.NewRuntimeConfig()
```

Comment:
I don't know what the Go convention is for it, but we should only be instantiating this once, ideally.  Probably as a (gag) global that gets instantiated once in `init()`.

Reply:
Ya I wasn't sure how to do that and had meant to come back to it when the config reader stuff came up.